sonar-plugin-api/src/main/java/org/sonar/api/server/ws/internal/ValidatingRequest.java

```diff
@@ -49,6 +51,15 @@ public void setAction(WebService.Action action) {
   }
 
   @Override
+  public LocalConnector getLocalConnector() {
```

Comment:
either missing a `@CheckForNull` or some check in the implementation is missing to enforce the method's contract